index.html

```diff
@@ -241,5 +241,66 @@ <h3>Skills</h3>
             </section>   
         </section>
    </section>
+   <section class="form-section">
+        <div class="form-background"></div>
+
+        <div class="get-started"> 
+
+        <h2>Get started</h2>
+    <p>
+      I'm always interested in hearing about <br />
+      new projects, so if you'd like to chat <br />
+      please get in touch.
+    </p>
+
+</div>
+
+
+
+
+
+       <section class="form">
+            <form class="form-view"  action="https://formspree.io/f/mlekzyqz" method="POST">
+               <div class="input-group">
+                
+                <label>
+                    Your Name:
+                </label>
+                <input type="text" name="email" placeholder="Anastasiia" maxlength="30" required>
```

Comment:
Good work creating a form tag 💯 

As this is a form, to have a relation between each input and it can be sent to your account on formspree, there are some that you need to change.

Here's the error when sending a form

![image](https://github.com/ctokoli/personal-portfolio-site/assets/93630700/69bca50a-4eb9-4457-9949-e750d85024e8)


- Kindly, in this `input` where the user has to put the email, you should put on the attribute name property equal to `name`, in this way the input will do a reference to different data in your form and it should work properly. You can read more about this on this [link](https://developer.mozilla.org/es/docs/Web/HTML/Element/form) 💯